tensorboard/webapp/app_routing/namespaced_state_reducer_helper.ts

```diff
@@ -85,10 +84,12 @@ export type NamespaceContextedState<
  * Utility for managing namespaced state. It returns namespace-contexted
  * `initialState` and `reducers` that help manage the namespaced state.
  *
- * An optional `onRouteKindOrExperimentsChanged` function allows more
- * fine-grained changes on state after a key change in Route - that is, after
- * navigating to a Route with either a change in route kind or in the set of
- * experiments. It guarantees that the change happens after namespaced state has
+ * An optional `onNavigated` function allows more fine-grained changes to state
+ * based on the routes information. This function gurantees in app navigation
```

Comment:
"This function gurantees in app navigation happens and has finished." - I think this sentence is redundant with the very next sentence. So probably could remove and keep the documentation a little shorter?